crates/iceberg/Cargo.toml

```diff
@@ -62,7 +63,8 @@ futures = { workspace = true }
 itertools = { workspace = true }
 murmur3 = { workspace = true }
 once_cell = { workspace = true }
-opendal = { workspace = true }
+# opendal = { workspace = true }
+opendal = { git = "https://github.com/apache/opendal", rev = "0a3e98f"}
```

Comment:
>[!NOTE]
>This is temporary so that I have this remotely against the changes made from https://github.com/apache/opendal/pull/4965. I'll remove this once a release has been made :+1:

Reply:
opendal plans to start release this week.